packages/oui-datagrid/src/util.js

```diff
@@ -1,11 +1,6 @@
+import get from "lodash/get";
+import isUndefined from "lodash/isUndefined";
+
 export function hasProperty (obj, prop = "") {
-    const props = prop.split(".");
-    let theObject = obj;
-    for (let i = 0; i < props.length; i++) {
-        if (!Object.prototype.hasOwnProperty.call(theObject, props[i])) {
-            return false;
-        }
-        theObject = theObject[props[i]] || {};
-    }
-    return true;
+    return !isUndefined(get(obj, prop));
```

Comment:
https://lodash.com/docs/4.17.15#has ?

Reply:
It doesn't have the same result because the checks made are not the same. `has` uses `Object.prototype.hasOwnProperty` which will result the same. 
Whereas `get` makes some checks by checking some `enumerable` property belongs to object (https://github.com/lodash/lodash/blob/master/.internal/isKey.js#L24)

Reply:
It doesn't have the same result because the checks made are not the same. `has` uses `Object.prototype.hasOwnProperty` which will result the same. 
Whereas `get` makes some checks by checking some `enumerable` property belongs to object (https://github.com/lodash/lodash/blob/master/.internal/isKey.js#L24)

Reply:
@marie-j and what about https://lodash.com/docs/4.17.15#hasIn ? it looks the same : https://github.com/lodash/lodash/blob/master/hasIn.js

Reply:
@frenautvh did not see that one ! I'll try it out and update if it's the case